lib/lte_link_control/lte_lc.c

```diff
@@ -441,8 +441,9 @@ static int enable_notifications(void)
 		/* %XT3412 notifications subscribe */
 		err = at_cmd_write(buf_sub, NULL, 0, NULL);
 		if (err) {
-			LOG_ERR("Failed to subscribe to XT3412 notifications");
-			return err;
+			LOG_WRN("%s failed (%d), TAU pre-warning notifications are not enabled",
+				buf_sub, err);
```

Comment:
I guess the same `log_strdup()` is needed here as well.

Reply:
Woops. Forgot that one.